src/client/refresh/gl1/gl1_model.c

```diff
@@ -166,14 +164,66 @@ Mod_Init(void)
 	memset(mod_novis, 0xff, sizeof(mod_novis));
 }
 
+/*
+=================
+Mod_AliasModelFixup
+=================
+*/
+static void
+Mod_AliasModelFixup(model_t *mod, const dmdl_t *pheader)
+{
+	mod->type = mod_alias;
+
+	if (pheader)
+	{
+		int i;
+
+		for (i=0 ; i<pheader->num_skins ; i++)
+		{
+			mod->skins[i] = R_FindImage((char *)pheader + pheader->ofs_skins + i*MAX_SKINNAME,
+				it_skin);
+		}
+	}
+
+	mod->mins[0] = -32;
+	mod->mins[1] = -32;
+	mod->mins[2] = -32;
+	mod->maxs[0] = 32;
+	mod->maxs[1] = 32;
+	mod->maxs[2] = 32;
+}
+
+/*
+=================
+Mod_SP2Fixup
+=================
+*/
+static void
+Mod_SP2Fixup(model_t *mod, const dsprite_t *sprout)
+{
+	mod->type = mod_sprite;
+
+	if (sprout)
+	{
+		int i;
+
+		/* byte swap everything */
```

Comment:
you're not byte-swapping anything here :-p

Reply:
Yep, my miss....